src/framework/components/render/component.js

```diff
@@ -682,7 +692,8 @@ class RenderComponent extends Component {
         });
     }
 
-    set materialAssets(value = []) {
+    set materialAssets(value) {
+        value = value || [];
```

Comment:
was this change intentional?

Reply:
Yeah, I thought this looked odd too. If there is some reason why default parameters shouldn't be used in a setter, I would at least recommend a comment because otherwise, somebody will just switch it back. :smile:

Reply:
to me based on looking online it seems ES6 does not support default parameters in setters.
Our lint flagged it as an issue.

Reply:
I tested it and the default value for setter works. When I assign undefined to it, it uses the default value instead. It seems the warning is only for TS, not for JS - can it be globally hidden?
<img width="587" alt="Screenshot 2021-11-05 at 09 12 47" src="https://user-images.githubusercontent.com/59932779/140486812-b4d70e35-be79-4a9e-b6b8-8d3fd791c589.png">